packages/odyssey-react-mui/src/OdysseyTranslationProvider.tsx

```diff
@@ -45,8 +45,17 @@ export const OdysseyTranslationProvider = ({
   translationOverrides,
 }: OdysseyTranslationProviderProps) => {
   useEffect(() => {
+    const changeHtmlLangAttr = () => {
+      window.document.documentElement.setAttribute(
+        "lang",
+        languageCode || "en"
```

Comment:
I would not set `lang` attribute as `languageCode`. `languageCode` is of type SupportedLanguages which uses `_` and that is not BCP 47 which violates best practices for the `lang` element

Similarly for `changeLanguage`, you might need to becareful as `window.navigator.langauge` is probably going to be BCP 47 but our internal framework uses our internal format with `_` instead of `-`.

Reply:
@queeniechen-okta Good call. I'll have to create some sort of formatting function to replace `_` with `-`. Thanks!